Source/Core/DolphinQt/Settings/PathPane.cpp

```diff
@@ -37,6 +37,8 @@ void PathPane::Browse()
       QFileDialog::getExistingDirectory(this, tr("Select a Directory"), QDir::currentPath()));
   if (!dir.isEmpty())
     Settings::Instance().AddPath(dir);
+  
+  add->setDown(false);
```

Comment:
This seems like a hack fix. Please see if connecting to the `clicked` event instead of `pressed` fixes this as I cannot reproduce the issue.

Reply:
The issue is also present on Windows aside from macOS. `clicked` will work but I didn't opt for that because `pressed` was used in multiple places for "Load" or "Open". It is also a UX thing since I have seen the Open button being pressed until the file dialog exits on macOS. Choosing `clicked` or `pressed` is your call.

Reply:
We are probably incorrectly using `pressed` everywhere and should be using `clicked` (which does still activate with keyboard input, btw).